src/lucky/tags/text_helpers.cr

```diff
@@ -0,0 +1,233 @@
+module Lucky::TextHelpers
+  @@_cycles = Hash(String, Cycle).new
+
+  def truncate(text : String, length : Int32 = 30, omission : String = "...", separator : String | Nil = nil, escape : Bool = false, blk : Nil | Proc = nil)
+    if text
+      content = truncate_text(text, length, omission, separator)
+      content = escape ? HTML.escape(content) : content
+      content += blk.call.to_s if blk.is_a?(Proc) && text.size > length
+      content
+    end
+  end
+
+  def truncate(text : String, length : Int32 = 30, omission : String = "...", separator : String | Nil = nil, escape : Bool = true, &block : -> _)
+    truncate(text, length, omission, separator, escape, blk: block)
+  end
+
+  # This could go in a String extension
```

Comment:
I agree. This should probably go in utils or in an optional string extension at some point. Can you remove the comment for now though?